backend/app/domain/services/base/model.py

```diff
@@ -489,3 +490,8 @@ def amount_of_models_uploaded_in_hr_diff(self, task_id: int, user_id: int):
             )
         )
         return amount_of_models_uploaded_in_hr_diff
+
+    def get_dynalab_model(self, task_code: str):
+        bucket = "https://models-dynalab.s3.eu-west-3.amazonaws.com"
```

Comment:
I'm sure there's a field in the database, in the task table, for the bucket variable.

Reply:
Not really for the dynalab model